.github/workflows/gfmrun.yml

```diff
@@ -0,0 +1,49 @@
+name: Test and Build
+
+on:
+  pull_request:
+    branches:
+      master
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  test:
+    name: Test GFMRUN
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v1
+        with:
+          ref: ${{ github.ref }}
+
+      - name: Set up Go 1.13
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
+
+      - name: Set GOPATH and PATH
+        run: |
+          echo "##[set-env name=GOPATH;]$(dirname $GITHUB_WORKSPACE)"
+          echo "##[add-path]$(dirname $GITHUB_WORKSPACE)/bin"
+        shell: bash
+
+      - name: Run Linter
+        run: |
+          curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin latest
+          golangci-lint run
```

Comment:
No action for this? Is it hard to make one?

Reply:
It's simply downloading and installing the linter binary and running it. Could you elaborate more on what do you want me to do here?